docs/sources/panels-visualizations/visualizations/datagrid/index.md

```diff
@@ -0,0 +1,117 @@
+  - ../../reference/datagrid/
+  - ../../visualizations/datagrid/
+description: Learn about datagrid panel visualization features.
+keywords:
+  - grafana
+  - dashboard
+  - panels
+  - datagrid
+  - datagrid panel
+  - datagrid options
+menuTitle: Datagrid
+title: Datagrid
+weight: 350
+---
+
+# Datagrid
+
+{{% admonition type="note" %}}
+
+The Grafana Datagrid panel is experimental. This feature is supported by the engineering team on a best-effort basis, and breaking changes may occur without notice prior to general availability.
+
+{{% /admonition %}}
+
+The Datagrid panel offers you the ability to create, edit, and fine-tune data within Grafana. As such, this panel can act as a data source for other panels
+inside a dashboard.
+
+Through it, you can manipulate data queried from any data source, or from a dragged and dropped file. You can then use the panel as a simple tabular
+visualization, or you can modify the data—and even remove it altogether—to create a blank slate.
+
+Editing the dataset changes the data source to use the inbuilt `-- Grafana --` data source, thus replacing the old data source settings and related queries, while also copying the current dataset into the dashboard model. You can either use a pre-existing dataset from any other data source, start with a blank slate, or you can drag and drop a file to create a new dataset.
+
+You can then use the panel as a data source for other panels, by using the inbuilt `-- Dashboard --` data source to pull the datagrid data. This allows for an interactive dashboard experience, where you can modify the data and see the changes reflected in other panels.
+
+Learn more about the inbuilt `-- Grafana --` and `-- Dashboard --` datasources in the [special data sources]({{< relref "../../../datasources/#special-data-sources"  >}}) documentation.
+
+![Datagrid panel](/media/docs/datagrid/screenshot-grafana-datagrid-panel.png)
+
+## Context menu
+
+To provide a more streamlined experience, the Datagrid panel has a context menu that can be accessed by right clicking on a cell, column header, or row selector. Depending on the state of your datagrid, the context menu offers different options including:
+
+- Delete or clear rows and columns.
+- Remove all existing data (rendering your datagrid blank).
+- Trigger search functionality, which allows you to find keywords within the dataset.
+
+Deleting a row or column will remove the data from the datagrid, while clearing a row or column will only remove the data from the cells, leaving the row or column intact.
+
+![Datagrid context menu](/media/docs/datagrid/screenshot-grafana-datagrid-context-menu.png)
+
+## Header menu
+
+You can also access a header menu by clicking the dropdown icon next to the header title. From here, you can not only delete or clear a column, but also rename it, freeze it, or convert the field type of the column.
+
+![Datagrid header menu](/media/docs/datagrid/screenshot-grafana-datagrid-header-menu.png)
```

Comment:
This is another image where the point of the image gets a bit lost. I'd suggest just cropping out the first three rows of the grid to give the context menu more focus. I know we'll lose the name of the panel, but that's fine.